src/index.html

```diff
@@ -11,25 +11,49 @@
   <body>
 
     <header class="header">
-      <div class="container">
-        <a href="index.html"> <img class="logo-img"
+
+      <a class="a" href="index.html"> <img class="logo-img"
```

Comment:
Using classes like `a`, `nav` or `li` is bad practice.

Class names should represent the meaning of the content (not the styles or tag names) [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)

![image](https://github.com/mate-academy/layout_moyo-header/assets/96138503/3ce9318c-2569-4080-8458-d60d04abae96)